docs/user/environment/3-further/other_apis.rst

```diff
@@ -0,0 +1,87 @@
+Connecting to other APIs
+========================
+
+Although we have covered an extensive list of supported atmospheric models,
+it is always possible to load data from other APIs.
+Indeed, it constitutes a significant part of the flexibility of RocketPy.
+
+
+With OPeNDAP protocol
+----------------------
+
+The OPeNDAP protocol allows you to access remote data in a simple and efficient way.
+We have used it to load data from the NOAA's data server in the :ref:`forecast`
+section.
+
+In case you want to use a different atmospheric model from NOAA, for example,
+you are going to need to provide the URL of corresponding API endpoint and a dictionary
+mapping the names of the variables to the names used in the dataset to the 
+RocketPy's Environment class.
+
+We use a dictionary because we need to determine the name used by the model of
+the following dimensions and variables:
+
+- Time
+- Latitude
+- Longitude
+- Pressure Levels
+- Geopotential Height (as a function of Time, Pressure Levels, Latitude and Longitude)
+- Surface Geopotential Height (as a function of Time, Latitude and Longitude)
+- Wind - U Component (as a function of Time, Pressure Levels, Latitude and Longitude)
+- Wind - V Component (as a function of Time, Pressure Levels, Latitude and Longitude)
+
+
+For example, let's imagine we want to use the HIRESW model from this endpoint: 
+`https://nomads.ncep.noaa.gov/dods/hiresw/ <https://nomads.ncep.noaa.gov/dods/hiresw/>`_
+
+
+Looking through the variable list in the link above, we find the following correspondence:
+
+- Time = "time"
+- Latitude = "lat"
+- Longitude = "lon"
+- Pressure Levels = "lev"
+- Geopotential Height = "hgtprs"
+- Surface Geopotential Height = "hgtsfc"
+- Wind - U Component = "ugrdprs"
+- Wind - V Component = "vgrdprs"
+
+Therefore, we can create an environment like this:
+
+.. code-block:: python
+
+    from rocketpy import Environment
+
+    env = Environment()
+
+    name_mapping = {
+       "time": "time",
+        "latitude": "lat",
+        "longitude": "lon",
+        "level": "lev",
+        "temperature": "tmpprs",
+        "surface_geopotential_height": "hgtsfc",
+        "geopotential_height": "hgtprs",
+        "u_wind": "ugrdprs",
+        "v_wind": "vgrdprs",
+    }
+
+    env.set_atmospheric_model(
+        type="forecast",
+        file="",
+        dictionary=name_mapping,
+    )
```

Comment:
I got an error while running this code block in a notebook

![image](https://github.com/user-attachments/assets/10eb8a36-41e6-4feb-96d5-be615cddc671)

Reply:
I added a message describing that this code block won't run correctly because the file argument is set to an empty string. I believe I've fixed the problem;